### PR TITLE
Added MCXAirPort manifest and icon

### PR DESCRIPTION
**Pull request by walkintom**

### CODE REVIEW

File: Manifests/ManagedPreferencesApple/com.apple.MCXAirPort.plist [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://apple.com/macos</string>
+	<key>pfm_description</key>
+	<string>AirPort settings</string>
+	<key>pfm_documentation_url</key>
+	<string>https://developer.apple.com</string>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApple/com.apple.MCXAirPort.plist]: @walkintom can you share or update the direct documentation links on the MCXRedirector & MCXAirPort manifests?  If others are going to contribute to these we'll need that info

Comment (wegotoeleven) [reply Manifests/ManagedPreferencesApple/com.apple.MCXAirPort.plist:10]: @apizz I just had a quick search, and I can only find a reference to the following:

**MCXRedirector**

https://github.com/jamf/Jamf-Nation-Managed-Preference-Manifests/blob/master/com.apple.MCXRedirector.manifest
https://mosen.github.io/profiledocs/payloads/mac/mcxredirector.html#payloadtype-com-apple-mcxredirector

**MCXAirPort**

https://github.com/jamf/Jamf-Nation-Managed-Preference-Manifests/blob/master/com.apple.MCXAirPort.manifest

I've tested the profile generated by the MCXAirPort and it works, but haven't tested MCXRedirector. I assumed that folder redirection was deprecated?

Comment (walkintom) [reply Manifests/ManagedPreferencesApple/com.apple.MCXAirPort.plist:10]: Must admit I completely forgot I added this - MCXRedirector _was_ apparently scheduled to be deprecated, but currently still works - sadly I can't find any documentation for it.

**Comment by apizz** [comment]
**Concern with MCXRedirector:** looking at https://github.com/jamf/Jamf-Nation-Managed-Preference-Manifests/blob/master/com.apple.MCXRedirector.manifest is that in this manifest there is a `pfm_name` key for the dictionary while in this PR this doesn't exist.

Before merging this, can you please test and see if this is needed?  Screenshots and the version(s) of macOS that this works for would also be helpful.

**Comment by kevinmcox** [comment]
Closing due to a lack of activity.

**Pull request closed without merging**